components/org.wso2.carbon.identity.sso.encryptor/src/main/java/Encryption.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
As a best practice, we do not create classes in the default package. We usually create a package unique for each component. Even though this is not a bundle, we need to follow this.